src/app/pages/AccountDetailsPage/index.tsx

```diff
@@ -67,6 +68,7 @@ export const AccountDetailsPage: FC = () => {
         token={token}
         tokenPriceInfo={tokenPriceInfo}
       />
+      <DappBanner scope={scope} address={address} />
```

Comment:
- https://b50d1300.oasis-explorer.pages.dev/mainnet/sapphire/address/0x8Bc2B030b299964eEfb5e1e0b36991352E56D2D3 shows the banner

- https://b50d1300.oasis-explorer.pages.dev/mainnet/sapphire/address/oasis1qpdgv5nv2dhxp4q897cgag6kgnm9qs0dccwnckuu doesn't